Header header
geometry_msgs/Vector3[] lane_points